ovh/provider.go

```diff
@@ -42,6 +42,7 @@ func Provider() *schema.Provider {
 		DataSourcesMap: map[string]*schema.Resource{
 			"ovh_cloud_region":                     dataSourceCloudRegion(),
 			"ovh_cloud_regions":                    dataSourceCloudRegions(),
+			"ovh_cloud_kubernetes_cluster":         dataSourceCloudKubernetesCluster(),
```

Comment:
Please keep it sorted
```suggestion
			"ovh_cloud_kubernetes_cluster":         dataSourceCloudKubernetesCluster(),
			"ovh_cloud_region":                     dataSourceCloudRegion(),
			"ovh_cloud_regions":                    dataSourceCloudRegions(),
```